tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -791,12 +791,18 @@ def fit(self,
             'excluded_model_types': excluded_model_types,
             'feature_prune_kwargs': kwargs.get('feature_prune_kwargs', None)
         }
+
+        aux_kwargs = {
+            'ag_args': ag_args,
+            'excluded_model_types': excluded_model_types,
+        }
```

Comment:
What are the ramifications of passing `ag_args` to aux models? Is this to enable HPO?